<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(172, 172, 172);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logoLoCarPequeno">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(172, 172, 172);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../AppData/PycharmProjects/LoCar/imagens/808107201585380077-64.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btVoltar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(165, 165, 165, 255));
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;

}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 248, 0, 255), stop:1 rgba(255, 134, 73, 255));
}

</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameConVeiculo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>631</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelNomeFunc">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelLoginFunc">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelMatriculaFunc">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btConsultarVeiculo">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Consultar veiculo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(165, 165, 165, 255));
border-style: outset;
border-width: 1px;
border-color: black;
border-radius: 5px;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(19, 226, 4, 255), stop:1 rgba(148, 209, 6, 255));
}


</string>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tbNumeroPlaca">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;</string>
     </property>
     <property name="placeholderText">
      <string>NÂº na Placa</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlacaVei">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelCorVei">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelModeloVei">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>130</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tagEnderecoCliente">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Placa:</string>
     </property>
    </widget>
    <widget class="QLabel" name="tagMarcaVeiculo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Marca:</string>
     </property>
    </widget>
    <widget class="QLabel" name="tagModeloVeiculo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modelo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="tagCNHCliente">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="tagAnoVeiculo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ano:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMarcaVei">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelTipoVei">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tagTelefoneCliente">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tipo: </string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAnoVei">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tagEnderecoCliente_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Chassi:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelChassiVei">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tagEnderecoCliente_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Valor DiÃ¡ria:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelValorDiariaVei">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btExcluir">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>350</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(165, 165, 165, 255));
border-radius: 5px;
border-style: outset;
border-width: 1px;
border-color: black;

}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 8, 255), stop:1 rgba(255, 0, 119, 255))
}

</string>
     </property>
     <property name="text">
      <string>Excluir</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
